docs/aws/Upgrading.rst

```diff
@@ -7,11 +7,17 @@ Upgrading an existing installation
    :local:
    :depth: 2
 
-If you already have an installation and you want to upgrade to the latest release, perform the following steps:
+Before starting an upgrade, read one of the following:
```

Comment:
This changes the meaning.
You need to read either or both depending on what you upgrade.